shrike/src/main/java/com/ibm/wala/shrike/shrikeBT/analysis/Analyzer.java

```diff
@@ -268,12 +268,9 @@ private void getReachableRecursive(
     if (from < 0) {
       throw new IllegalArgumentException("from < 0");
     }
-    while (true) {
-      // stop if we've already visited this instruction or if we've gone outside
-      // the mask
-      if (reachable.get(from) || (mask != null && !mask.get(from))) {
-        return;
-      }
+
+    // stop if we've already visited this instruction or if we've gone outside the mask
+    for (; !reachable.get(from) && (mask == null || mask.get(from)); ++from) {
```

Comment:
Would you be ok with a `while` loop here?  I find I have to think slightly harder about `for` loops without an initializer 🙂